sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -20,7 +21,10 @@ export interface AgentDeletionStatusOutput {
 
 // @public
 export interface AgentEventMessage {
-    data: AgentThreadOutput | ThreadRunOutput | RunStepOutput | ThreadMessageOutput | MessageDeltaChunk | RunStepDeltaChunk | string;
+    // Warning: (ae-forgotten-export) The symbol "AgentThreadOutput_2" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "RunStepOutput_2" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "ThreadMessageOutput_2" needs to be exported by the entry point index.d.ts
+    data: AgentThreadOutput_2 | ThreadRunOutput | RunStepOutput_2 | ThreadMessageOutput_2 | MessageDeltaChunk | RunStepDeltaChunk | string;
```

Comment:
the _2 stuff shows up when you reference two interfaces with the same name, but I think you should be able to export your public versions of these interfaces and not the wire versions, right?

Reply:
That is correct. I'll address all the _2 in a separate PR